tensorflow/core/distributed_runtime/eager/eager_service_impl.h

```diff
@@ -148,9 +148,9 @@ class EagerServiceImpl {
 
     bool IsStale() {
       mutex_lock l(last_accessed_mu_);
-      return (destroy_after_micros_ > 0 &&
-              (env_->env->NowMicros() - last_accessed_micros_) >
-                  destroy_after_micros_);
+      const int64 time_passed = env_->env->NowMicros()
+                                - last_accessed_micros_);
```

Comment:
This can also go on the same line, I think

Reply:
updated.

Reply:
updated.